Sources/Subs-Db-mysql.php

```diff
@@ -272,7 +272,7 @@ function smf_db_replacement__callback($matches)
 
 		case 'identifier':
 			// Backticks inside identifiers are supported as of MySQL 4.1. We don't need them for SMF.
-			return '`' . strtr($replacement, array('`' => '', '.' => '')) . '`';
```

Comment:
I feel that the `itentifier` filter should either accept table.field syntax, or return an error - instead of just mangling the input. A couple sanitization libraries I've used seem to accept table.field.